### ISSUE TITLE
Report a problem not consistent with Canada.ca theme

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#109]
URL: Any page
User Agent: Chrome
Language: Both

Issue Description: The 'Report a problem or mistake on this page' button has been implemented differently than the Canada.ca pattern.
For details refer to this URL: https://design.canada.ca/common-design-patterns/report-problem.html

Please note that the button, the label, the placement and the privacy statement are mandatory. 
![image](https://user-images.githubusercontent.com/68117281/137014441-6a73d93c-2273-4c4b-8633-fbff75d4547f.png)

Steps to reproduce:
Expected results: The 'Report a problem' should be consistent with that of the Canada.ca theme template.  Please refer to https://www.canada.ca/en.html

**Comment by ndolter** [comment]
Re-tested and the Report a problem does not appear to be updated yet.
![image](https://user-images.githubusercontent.com/68117281/137382161-d510f12a-6766-4949-9dc0-341208776f63.png)

**Comment by echouin** [comment]
Hi @ndolter this is not an accessibility issue, so it has been closed on the OP side.

**Comment by echouin** [comment]
@ndolter Additional details: we will look to implementing this revision post-MVP.

**Comment by echouin** [comment]
@mmiron-HC The final release has the following to meet the needs of our users in reporting website issues:
![image](https://user-images.githubusercontent.com/88099454/148396194-d6697f63-3b36-4dfc-bba0-0e3a36f273e8.png)

**Comment by ndolter** [comment]
Closing as this is no longer an issue.
